course/templates/course/index.html

```diff
@@ -16,7 +16,7 @@ <h1>{{ welcome_text|safe }}</h1>
 	<div class="frontpage-panel">
 		<h2 class="panel-heading">{% trans "My courses" %}</h2>
 		{% if my_instances %}
-			<div class="row frontpage">
+			<div class="frontpage frontpage-section">
```

Comment:
I wonder if [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) would make more sense semantically here?

Reply:
Good point, did not thought of that. Though the `<section>`-tag is not used elsewhere in the code... What do you think of using different tags even though the rest of the code is not using them? If it is ok, I could apply it after I hear I reviews and possible change-ideas of others.

Reply:
> What do you think of using different tags even though the rest of the code is not using them?

If they make the site better, feel free to make the site better :)

Reply:
That's fine!

Reply:
Like Matthew has suggested earlier, there are small accessibility issues that can be fixed when someone is working on the relevant code. For example, fixing the semantic markup for sections fits here.